src/RokuAdapter.ts

```diff
@@ -792,7 +812,7 @@ export class RokuAdapter {
                     return children;
                 }
                 let match;
-                match = /(\w+):(.+)/i.exec(line);
+                match = /(\S+[^:]):(.+)/i.exec(line);
```

Comment:
This adds support for keys like this:
```
"Food & Both":{...},
"Food&And":{...},
"Food Space":{...}
```

Reply:
What about keys with colons in them? 

`"my:key"`

Reply:
While possible I'm not sure how we could detect that because of the way the Roku prints key to the console.